### ISSUE TITLE
localeapp.com does not redirect to www.localeapp.com

### ISSUE THREAD

**Issue by tywhang** [Locale/localeapp#260]
Can somebody configure the DNS settings to redirect to www when it's missing.

I spent quite a long time wondering why I wasn't able to use the site.

Should save some time for others.

**Comment by michaelbaudino** [comment]
Indeed, we migrated our DNS infrastructure recently, this edge case seems to not work with HTTPS anymore (it does work with HTTP). We'll fix this as soon as possible and keep you posted.

Thanks for reporting it :tophat:
